pandas/core/base.py

```diff
@@ -1154,32 +1154,27 @@ def factorize(self, sort=False, na_sentinel=-1):
         dtype: int64
 
         >>> x.searchsorted(4)
-        array([3])
+        array([3], dtype=int64)
```

Comment:
Is there a reason you are adding the 'dtype=int64' here? I don't see that in the output of my console.

Reply:
It appears in my console, but I already delete them in 6ffdac79a